### PR TITLE
Drop dependency on kitchen

### PR DESCRIPTION
**Pull request by Flowdalic**
Kitchen is unmainted and should no longer be used, hence we this drops the dependency on kitchen.

### CODE REVIEW

File: xml2rfc/writers/text.py [hunk 1]
```diff
@@ -1797,8 +1796,10 @@ def join_cols(left, right):
             for i in range(t):
                 l = left[i]
                 r = right[i]
-                #assert displength(l)+displength(r)<70
-                w = 72-displength(l)-displength(r)
+                textwidth_l_ = textwidth(l)
```
Comment (kesara) [inline xml2rfc/writers/text.py:1799]: Typo here: `textwidth_l_` should be `textwidth_l`

**Review by kesara — CHANGES_REQUESTED** [review]

**Comment by kesara** [comment]
Looks like this change needs a bit more tweaking, running tests generated following differences.
```
diff --git a/tests/valid/elements.bom.text b/tests/valid/elements.bom.text
index 11611ce8..23f6df99 100644
--- a/tests/valid/elements.bom.text
+++ b/tests/valid/elements.bom.text
@@ -5,7 +5,7 @@
 Network Working Group                                          A. Author
 Internet-Draft                                        רוני אבן (R. Even)
 Obsoletes: 1234, 5678, 9012, 3456, 7890 (if              וואווי (Huawei)
-           approved)                             田中花子 様 (H. Tanaka)
+           approved)                                  田中花子 様 (H. Tanaka)
 Intended status: Informational
 Expires: 13 January 2019                                          N. Org

diff --git a/tests/valid/elements.pages.text b/tests/valid/elements.pages.text
index 88b8d591..0e1bd885 100644
--- a/tests/valid/elements.pages.text
+++ b/tests/valid/elements.pages.text
@@ -5,7 +5,7 @@
 Network Working Group                                          A. Author
 Internet-Draft                                        רוני אבן (R. Even)
 Obsoletes: 1234, 5678, 9012, 3456, 7890 (if              וואווי (Huawei)
-           approved)                             田中花子 様 (H. Tanaka)
+           approved)                                  田中花子 様 (H. Tanaka)
 Intended status: Informational
 Expires: January 13, 2019                                         N. Org

diff --git a/tests/valid/elements.text b/tests/valid/elements.text
index b71d95da..f1113779 100644
--- a/tests/valid/elements.text
+++ b/tests/valid/elements.text
@@ -5,7 +5,7 @@
 Network Working Group                                          A. Author
 Internet-Draft                                        רוני אבן (R. Even)
 Obsoletes: 1234, 5678, 9012, 3456, 7890 (if              וואווי (Huawei)
-           approved)                             田中花子 様 (H. Tanaka)
+           approved)                                  田中花子 様 (H. Tanaka)
 Intended status: Informational
 Expires: January 13, 2019                                         N. Org

diff --git a/tests/valid/elements.wip.text b/tests/valid/elements.wip.text
index 452a6c9f..30ee2238 100644
--- a/tests/valid/elements.wip.text
+++ b/tests/valid/elements.wip.text
@@ -5,7 +5,7 @@
 Network Working Group                                          A. Author
 Internet-Draft                                        רוני אבן (R. Even)
 Obsoletes: 1234, 5678, 9012, 3456, 7890 (if              וואווי (Huawei)
-           approved)                             田中花子 様 (H. Tanaka)
+           approved)                                  田中花子 様 (H. Tanaka)
 Intended status: Informational
 Expires: 13 January 2019                                          N. Org
```

An easy way to test this is:
```
xml2rfc --allow-local-file-access tests/input/elements.xml && head tests/input/elements.txt
```

**Comment by kesara** [comment]
@Flowdalic Actually, https://github.com/ietf-tools/xml2rfc/pull/914 fixes the above issue. Thanks.

**Review by kesara — APPROVED** [review]

**Comment by Flowdalic** [comment]
Thanks, would be great to see an xml2rfc release without a dependency on kitchen. That would help https://github.com/gentoo/gentoo/pull/27749 to progress.

**Comment by kesara** [comment]
These changes will be in the next xml2rfc release.
Most probably that will be in next week.

**Pull request merged**